modules/serverless-create-kafka-namespaced-broker.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * serverless/eventing/brokers/kafka-broker.adoc
+
+:_content-type: CONCEPT
+[id="serverless-create-kafka-namespaced-broker-{context}"]
+= Creating a Knative broker for Apache Kafka that uses an isolated data plane
+
+:FeatureName: The Knative Broker implementation for Apache Kafka with isolated data plane
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+To create a `KafkaNamespaced` broker, you must set the `eventing.knative.dev/broker.class` annotation to `KafkaNamespaced`.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource are installed on your {product-title} cluster.
+
+* You have access to an Apache Kafka instance such as link:https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/amq_streams_on_openshift_overview/kafka-concepts_str#kafka-concepts-key_str[Red Hat AMQ Streams], and have created a Kafka topic.
+
+* You have created a project, or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create an Apache Kafka-based broker by using a YAML file:
++
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    eventing.knative.dev/broker.class: KafkaNamespaced <1>
+  name: default
+  namespace: my-namespace <2>
+spec:
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: my-config <2>
+    # namespace: my-namespace <3>
+...
+----
+<1> To use the Apache Kafka broker with isolated data planes, the broker class value must be `KafkaNamespaced`.
+<2> The referenced `ConfigMap` object `my-config` must be in the same namespace as the `Broker` object, in this case `my-namespace`.
+<3> There is no need to define the namespace for the ConfigMap object  because it defaults to the namespace of the `Broker` object.
+
+. Apply the Apache Kafka-based broker YAML file:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+[IMPORTANT]
+====
+The `ConfigMap` object that is specified in `spec.config` must be in the same namespace as the `Broker` object:
```

Comment:
```suggestion
The `ConfigMap` object specified in `spec.config` must be in the same namespace as the `Broker` object:
```

Reply:
I got rid of both to leave "The `ConfigMap` object in `spec.config` "